test/e2e-shared-tests/helpers.go

```diff
@@ -77,58 +73,6 @@ func waitForConfigDriftMonitorStart(t *testing.T, cs *framework.ClientSet, node
 	require.Nil(t, err, "expected config drift monitor to start (waited %v)", end)
 }
 
-func assertNodeAndMCPIsDegraded(t *testing.T, cs *framework.ClientSet, node corev1.Node, mcp mcfgv1.MachineConfigPool, filename string) {
```

Comment:
I see you've moved a lot of these from helpers to the mcd drift test, was it because they weren't being used elsewhere?

Reply:
Just for posterity: I put these in the `helpers.go` file just to keep the `mcd_config_drift.go` file clean. But I'm OK with this change.